colorful/experimental-theme.json

```diff
@@ -0,0 +1,88 @@
+{
+    "global": {
+        "presets": {
+            "color": [
+                {
+                    "slug": "valencia",
+                    "value": "#DB403B"
+                },
+                {
+                    "slug": "mustard",
+                    "value": "#FFDE69"
+                },
+                {
+                    "slug": "navy",
```

Comment:
@jffng Could you help me understand why a slug is needed here? If the slug is `navy` and that is then used as part of a CSS variable name, I feel like that might defeat the purpose of a CSS variable. To me, the purpose of a variable is that the value van vary. If the value (`navy` or `022384`) is used in the name, you can't change the value of that variable without the name becoming redundant. Once I change the value of `--wp--preset--color--navy` to something other than `#022384`, the word "navy" no longer describes the value. 

That makes me wonder if slugs should be completely removed, and automatically generated to create a predictable standard which themes could rely on their stylesheets. They could be named based on their order, for example:

`--wp--preset--color--one`
`--wp--preset--color--two`
`--wp--preset--color--three`

If I am totally misunderstanding something, please let me know!

Reply:
Totally agree. I’ve gone with primary, secondary, etc in the Go theme (for this same reason).

Reply:
We also had an issue recently where a popular plugin overwrote the primary and secondary color values, which is often not what we want?

Reply:
Yeah, for now the slug is only used to generate a CSS variable name, so something more generic would make sense. 

The exact names here relate back to this old issue around whether or not to standardize custom color names: https://github.com/WordPress/gutenberg/issues/7553

Some standardization does make sense, and the advent of Global Styles may be a good time to start.

Reply:
@johnstonphilip thanks for the review! 

Re: slugs, that's a fair point. In the case of colors, they could be something generic. I named them as such because it made the example more straightforward to keep my naming straight between the editor, the theme's stylesheet, and the theme config.

> That makes me wonder if slugs should be completely removed, and automatically generated to create a predictable standard which themes could rely on their stylesheets

I think I agree. Right now there is no relationship between `presets` and `styles` in the theme config. It could make sense if there were a set of common CSS variables that Gutenberg used to predictably style core blocks, and themes could override.

Reply:
>popular plugin overwrote the primary and secondary color values

@carolinan out of curiosity, what do you mean? The theme would provide some CSS variables, and an unrelated plugin would overwrite them because they happened to be named the same?

Reply:
Yes, like this: https://github.com/Yoast/wordpress-seo/issues/15345